### PR TITLE
Tooltip light theme support

### LINKED ISSUE #396: tooltip: add support for light theme
### Desired behavior

I think we should make the entire tooltip dark when used in Light Theme. If for some reason we don't want to do that, the tail at least needs more contrast when placed on the top.

### Today's behavior

screen shot 2018-12-13 at 9 49 28 am

### LINKED ISSUE #397: tooltip: can see entire diamond when placed on bottom
### Expected behavior

When tail is on bottom, should only see the bottom of the diamond that makes the tail.

### Actual behavior

Can see top of diamond too.

screen shot 2018-12-13 at 9 54 15 am

screen shot 2018-12-13 at 9 54 53 am

### Steps to repro

1. Use tooltip with tail placement anywhere on the bottom. Probably need a high-resolution monitor. I'm on a MacBook with a 2880 x 1800 screen.

### Related packages

- tooltip@1.3.19

### Environment

- OS - MacOS 10.14
- Browser version - Chrome v70
- Screen Resolution: 2880 x 1800

### PR DESCRIPTION
**Pull request by jaketrent**
Fixes #396 
Fixes #397

### CODE REVIEW

File: packages/tooltip/src/css/index.js [hunk 1]
```diff
@@ -36,51 +41,87 @@ export default {
     color: core.colors.white,
     backgroundColor: core.colors.blue
   },
-  // --tailPosition
-  [`.psds-tooltip:after`]: {
+
+  [`.psds-tooltip__tail`]: {
```
Comment (danethurber) [inline packages/tooltip/src/css/index.js]: i'm seeing some clipping of the tail shadow - on the left

screen shot 2019-01-04 at 9 05 26 am

i wonder if you can make the width a bit larger to give the shadow some room and center the arrow

Comment (jaketrent) [reply packages/tooltip/src/css/index.js]: :done:, now there is a box shadow blur width on both sides of the overflow-hidden parent container.

**Comment by jaketrent** [comment]
@danethurber this is ready for another look, please.

**Comment by danethurber** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>:exclamation: Please assign a teammate to review your PR.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9a1429467fe6a9eef938f40a0342154af6dfff41
</p>

**Comment by danethurber** [comment]
great work!

**Pull request merged**